cmd/wallet-web/src/pages/Store.vue

```diff
@@ -158,7 +158,7 @@ export default {
       processedCredentials: [],
       errors: [],
       vaults: [],
-      selectedVault: '',
+      setSelectedVault: '',
```

Comment:
Why did you rename your data variable instead of a method? This should stay as it used to be - `selectedVault`, it is not a function

Reply:
shoot i misread your comment , updated now